### ISSUE TITLE
Assumption that /bin/sh -> bash

### ISSUE THREAD

**Issue by iostanin-quantenna** [ericmartel/Sublime-Text-2-Perforce-Plugin#40]
When constructing commands it is assumed that the shell that will execute them is bash. In particular I mean using 'source' command instead of '.'. On ubuntu /bin/sh points to dash and soursing environment file does not work.

**Comment by ericmartel** [comment]
Hi Ilya,

Is it possible that you have changes on your system causing this?  The  plugin was initially tested on Ubuntu and worked without a problem.

Cheers

**Comment by iostanin-quantenna** [comment]
Hi Eric,

Can't think of any changes that might have been the cause. I just saw this message in the logs:

```
Perforce [warning]: /bin/sh: 1: source: not found
Perforce: File is not under the client root.
```

Then there is line 60:

`command_prefix = 'source ' + p4Env + ' && '`

Python subprocess docs says that it uses /bin/sh by default. And in Ubuntu (and Debian) /bin/sh -> dash. After I changed `source` to `.` everything worked fine.

Sorry just realized that I'm using ST3 version, but the issue is still the same.
